src/GrpcProtos/capture.proto

```diff
@@ -33,7 +33,8 @@ message InstrumentedFunction {
 message ApiFunction {
   string module_path = 1;
   string module_build_id = 2;
-  uint64 address = 3;
+  uint64 relative_virtual_address = 3;
```

Comment:
I changed because it's more descriptive, "address" is really generic. I initially used the "address" field as an absolute address only to realize it was actually a relative address. I prefer keeping as is.